MvvmCross/Core/Core/ViewModels/IMvxViewModel.cs

```diff
@@ -37,14 +37,13 @@ public interface IMvxViewModel
 
     public interface IMvxViewModel<TParameter> : IMvxViewModel where TParameter : class
     {
-        Task Initialize(TParameter parameter);
+        void Declare(TParameter parameter);
```

Comment:
@martijn00 so this PR introduces a new ViewModel lifecyle method? It isn't in the PR description/any new docs 🤔